modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -339,6 +339,14 @@ private DockerBuildImage _addTaskBuildDockerImage(
 		DirectoryProperty inputDirectoryProperty =
 			dockerBuildImage.getInputDir();
 
+		Property<Boolean> pull = dockerBuildImage.getPull();
+
+		pull.set(true);
+
+		Property<Boolean> noCache = dockerBuildImage.getNoCache();
```

Comment:
Can we just se "pull" to true and leave "noCache" to false?  Does that help with the problem of seeing the new update? 

Were you able to reproduce the problem where a new liferay "updated" image is not seen with our "buildDockerImage" task?  WE need a failing test case first before we can be sure we can fix the problem.

Can you send me the steps of how to reproduce the problem? of new docker image (that has been updated from a tag) doesn't get pulled?

Reply:
Hi greg, I sent test steps to you in mail.